index.bs

```diff
@@ -684,7 +684,9 @@ The initial value of ModuleStatus.prototype.constructor is the intrinsic object
 1. Let _entry_ be *this* value.
 1. If Type(_entry_) is not Object, throw a *TypeError* exception.
 1. If _entry_ does not have all of the internal slots of a ModuleStatus Instance (<a href="#module-status-internal-slots">5.5</a>), throw a *TypeError* exception.
-1. Return _entry_.[[Module]].
+1. Let _module_ be _entry_.[[Module]].
```

Comment:
this was a bug, we should never expose entry.[[Module]], instead, a namespace is what users can consume. with this change, you can inspect any entry, extract their namespace, and use it when creating a reflective module record that re-export from an existing namespace/module.